lib/taskgraph.js

```diff
@@ -20,7 +20,7 @@ const TASKGRAPH_PATHS = [
 ];
 
 
-const TREEHERDER_ROUTE = 'treeherder';
+const TREEHERDER_ROUTE = 'tc-treeherder';
```

Comment:
We should create a new route config here looks like: https://github.com/mozilla/autolander/blob/master/config/production.js#L42

If I understand correctly, production should use 'tc-treeherder', and for running tests on stage we should use 'tc-treeherder-staging'.